src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -0,0 +1,425 @@
+
+If you have any questions related to this feature, feel free to contact us at `markus@sentry.io`.
+
+## A basic example
+
+Say you have an exception message which, unfortunately, contains IP addresses which are not supposed to be there. You'd write:
+
+```json
+{
+  "applications": {
+    "$string": ["@ip:replace"]
+  }
+}
+```
+
+It reads as "replace all IP addresses in all strings", or "apply `@ip:replace` to all `$string` fields".
+
+`@ip:replace` is called a rule, and `$string` is a [_selector_](#selectors).
+
+## Built-in rules
+
+The following rules exist by default:
+
+- `@ip:replace` and `@ip:hash` for replacing IP addresses.
+- `@imei:replace` and `@imei:hash` for replacing IMEIs
+- `@mac:replace`, `@mac:mask` and `@mac:hash` for matching MAC addresses
+- `@email:mask`, `@email:replace` and `@email:hash` for matching email addresses
+- `@creditcard:mask`, `@creditcard:replace` and `@creditcard:hash` for matching creditcard numbers
+- `@userpath:replace` and `@userpath:hash` for matching local paths (e.g. `C:/Users/foo/`)
+- `@password:remove` for removing passwords. In this case we're pattern matching against the field's key, whether it contains `password`, `credentials` or similar strings.
+- `@anything:remove`, `@anything:replace` and `@anything:hash` for removing, replacing or hashing any value. It is essentially equivalent to a wildcard-regex, but it will also match much more than strings.
+
+## Writing your own rules
+
+Rules generally consist of two parts:
+
+- *Rule types* describe what to match. See [_Rule Types_](#rule-types) for an exhaustive list.
+- *Rule redaction methods* describe what to do with the match. See [_Redaction Methods_](#redaction-methods) for a list.
+
+Each page comes with examples. Try those examples out by pasting them into the "PII config" column of [_Piinguin_](https://getsentry.github.io/piinguin) and clicking on fields to get suggestions.
+
+## Selectors
+
+You have the possibility to select a region of the event using JSON-path-like syntax. To delete a specific key in "Additional Data", you could for example use:
+
+```json
+{
+  "applications": {
+    "extra.foo": ["@anything:remove"]
+  }
+}
+```
+
+### Boolean logic
+
+You can combine selectors using boolean logic.
+
+* Prefix with `!` to invert the selector. `foo` matches the JSON key `foo`, while `(~foo)` matches everything but `foo`.
+* Build the conjunction (AND) using `&&`, such as: `foo && !extra.foo` to match the key `foo` except when inside of `extra`.
+* Build the disjunction (OR) using `||`, such as: `foo || bar` to match `foo` or `bar`.
+
+### Wildcards
+
+* `**` matches all subpaths, so that `foo.**` matches all JSON keys within `foo`.
+* `*` matches a single path item, so that `foo.*` matches all JSON keys one level below `foo`.
+
+### Value types
+
+The following can be used to select subsections by JSON-type or semantic meaning.
```

Comment:
Helpful to remove passive voice where possible, suggest "Select subsections by JSON-type or semantic meaning using the following:"